bot/cogs/token_remover.py

```diff
@@ -86,6 +75,24 @@ def mod_log(self) -> ModLog:
                 channel_id=Channels.mod_alerts,
             )
 
+    @classmethod
+    def is_token_in_message(self, msg: Message) -> bool:
```

Comment:
I was expecting flake8 to nag here, but I guess there's no lint for it.

Via [PEP 8, the first argument to a classmethod should be named `cls`](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments)

Reply:
will be done, thanks